components/core/src/clp/regex_utils/regex_translation_utils.cpp

```diff
@@ -37,6 +37,7 @@ class TranslatorState {
     enum class RegexPatternState : uint8_t {
         NORMAL = 0,
         DOT,
+        ESCAPED,
         END,
```

Comment:
I missed this in the last PR, but enum constants should be CamelCase (so `Normal`, `Dot`, etc.).
I can't make a suggestion since this PR didn't touch all the lines, so you'll have to fix it manually. Sorry.